CHANGELOG.md

```diff
@@ -22,7 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `GET /.well-known/openeo` and `GET /`: `production` fields default to `false` instead of `true`.
 - `GET /collections` and `GET /collections/{collection_id}`: Additional dimensions in `cube:properties` can only be of type `other`.
 - Allow all STAC versions that are compatible to STAC 0.9.0.
-- Process graph nodes have an additional field `namespace` to distinguish pre-defined and user-defined processes. The old behavior to prioritize user-defined over pre-defined processes has been removed. [#305](https://github.com/Open-EO/openeo-api/issues/305)
+- Process graph nodes have an additional field `namespace` to distinguish pre-defined and user-defined processes. The old behavior to prioritize user-defined over pre-defined processes is still the default. [#305](https://github.com/Open-EO/openeo-api/issues/305)
```

Comment:
> The old behavior to prioritize user-defined over pre-defined processes is still the default.

->
 When no namespace is given explicitly: user-defined processes have priority over predefined process (the old behavior)

Reply:
I think I'm going for: The default behavior still prioritizes user-defined over pre-defined processes.

Reply:
The point of my suggestion was mainly to add `When no namespace is given explicitly` somewhere in there. I know it is obvious for us, actors in this PR, but maybe less obvious for others.

Reply:
Hmm... is "namespace = null" a "given namespace"?

Reply:
I think for a CHANGELOG entry it shouldn't be very pedantic about that

Reply:
> When the user didn't specify a namespace ...

Reply:
That doesn't quite express what I intend to say there. I want to express that this is more an addition, not a breaking change (the default behavior is still the same). That is somewhat lost...